

export const fail = async (req, res) => {
    throw Error;
};